app/core/utils/hash_util.py

```diff
@@ -4,10 +4,12 @@
 
 
 class WidgetInstanceHash:
```

Comment:
Given we're calling the file `hash_util`, let's just rename the class to a more generic `HashUtil` as opposed to calling it specifically `WidgetInstanceHash`